WalletWasabi.Bench/TransactionProcessorBench.cs

```diff
@@ -0,0 +1,138 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using BenchmarkDotNet.Attributes;
+using NBitcoin;
+using WalletWasabi.KeyManagement;
+using WalletWasabi.Models;
+using WalletWasabi.Services;
+using System.IO;
+using WalletWasabi.Helpers;
+using System.Runtime.CompilerServices;
+using NBitcoin.Protocol;
+using WalletWasabi.Stores;
+using System.Net;
+
+namespace WalletWasabi.Bench
+{
+	[ClrJob(baseline: true), CoreJob]
```

Comment:
This is new for me pls add more comments.

Reply:
This is copy/pasted. I didn't invest much time trying to understand it. But it means that we want to run this job for both Clr and Core, it could be mono, CoreRT and some other. 

I have removed it. Thank you.